swift-ci/master/centos/8/Dockerfile

```diff
@@ -30,6 +30,7 @@ RUN yum install --enablerepo=PowerTools -y \
     python3-six           \
     python3-pexpect       \
     platform-python-devel \
+    rpm-build.x86_64      \
```

Comment:
hi @discogestalt we should remove  rpm-build package from these docker images - we do not intend to create RPMs and want to keep the list minimal

Reply:
Ok, revision coming up.